src/SignalRServiceExtension/TriggerBindings/Resolver/DefaultRequestResolver.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Azure.SignalR.Serverless.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class DefaultRequestResolver : IRequestResolver
+    {
+        public bool ValidateContentType(HttpRequestMessage request)
+        {
+            var contentType = request.Content.Headers.ContentType.MediaType;
+            if (string.IsNullOrEmpty(contentType))
+            {
+                return false;
+            }
+            return contentType == Constants.JsonContentType || contentType == Constants.MessagePackContentType;
+        }
+
+        public bool ValidateSignature(HttpRequestMessage request, string accessToken)
+        {
+            //TODO: Add real signature validation
+            return true;
+        }
+
+        public bool TryGetInvocationContext(HttpRequestMessage request, out InvocationContext context)
+        {
+            context = new InvocationContext();
+            // Required properties
+            context.ConnectionId = request.Headers.GetValues(Constants.AsrsConnectionIdHeader).FirstOrDefault();
+            context.Hub = request.Headers.GetValues(Constants.AsrsHubNameHeader).FirstOrDefault();
+            context.Category = request.Headers.GetValues(Constants.AsrsCategory).FirstOrDefault();
+            context.Event = request.Headers.GetValues(Constants.AsrsEvent).FirstOrDefault();
+            if (string.IsNullOrEmpty(context.ConnectionId) ||
+                string.IsNullOrEmpty(context.Hub) ||
+                string.IsNullOrEmpty(context.Category) ||
+                string.IsNullOrEmpty(context.Event))
```

Comment:
Some checks already covered in `SignalRTriggerDispater.ExecuteAsync()`, could be reduced here?